sys/net/network_layer/ipv6/addr/ipv6_addr_from_str.c

```diff
@@ -163,6 +163,31 @@ ipv6_addr_t *ipv6_addr_from_str(ipv6_addr_t *result, const char *addr)
     return ipv6_addr_from_buf(result, addr, strlen(addr));
 }
 
+int ipv6_prefix_from_str(ipv6_addr_t *result, const char *addr)
+{
+    size_t str_len;
+    int pfx_len;
+    const char *separator;
+
+    if ((result == NULL) || (addr == NULL)) {
+        return -1;
+    }
+
+    if ((separator = strrchr(addr, '/')) == NULL) {
+        str_len = strlen(addr);
+        pfx_len = 128;
+    } else {
+        str_len = separator - addr;
+        pfx_len = atoi(separator + 1);
```

Comment:
Since 0 is a valid length for a prefix (for the default route), how about using `strtol` instead?